Source/Renderer/UniformState.js

```diff
@@ -261,7 +261,7 @@ define([
          * @memberof UniformState.prototype
          * @private
          */
-        inverseTranposeModel : {
+        inverseTransposeModel : {
```

Comment:
Actually I realized this is done already in https://github.com/AnalyticalGraphicsInc/cesium/pull/4115 which hasn't been merged yet.